src/chain/chain.rs

```diff
@@ -20,7 +20,7 @@ use crate::{
     parsec::{DkgResult, DkgResultWrapper},
     relocation::{self, RelocateDetails},
     utils::LogIdent,
-    BlsPublicKeySet, BlsSecretKeyShare, BlsSignature, ConnectionInfo, Prefix, XorName, Xorable,
```

Comment:
I thought we would need to pass at least signature back to vault, and possibly public key at some point?

Reply:
There is no API currently that uses these. But we can always expose them later, when they are actually needed.